keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/UnknownBaseImage.kt

```diff
@@ -5,3 +5,6 @@ import com.netflix.spinnaker.kork.exceptions.SystemException
 
 class UnknownBaseImage(os: String, label: BaseLabel) :
   SystemException("Could not identify base image for os $os and label $label")
+
+class UnknownBaseImageId(id: String):
+  SystemException("Could not identify base image for id $id")
```

Comment:
Yes! My convention of not smurf-naming exception types is catching on!

Reply:
What is smurf naming??

Reply:
My guess is that this refers to having `Exception` at the end of an exception class name (e.g., `ThisIsBrokenException`) vs just calling it `ThisIsBroken`.